itn-installer.sh

```diff
@@ -78,7 +78,8 @@ systemctl enable rusk
 systemctl daemon-reload
 
 echo "Setup local firewall"
-ufw allow 9000:9005/udp
+ufw allow 8080/tcp
+ufw allow 9000/udp
```

Comment:
Now the local firewall is in sync with the documentation. LMK if that's not a desired change.